plugins/node.d/hddtemp_smartctl

```diff
@@ -187,6 +187,10 @@ if ($^O eq 'linux') {
   opendir(DEV, '/dev');
   @drives = grep /^(ada?|da)[0-9]+$/, readdir DEV;
   closedir(DEV);
+} elsif ($^O eq 'midnightbsd') {
```

Comment:
This is the same as above - maybe just use `} elsif (($^O eq 'freebsd') || ($^O eq 'freebsd')) {`.

Every line counts :)